scripts/script.js

```diff
@@ -0,0 +1,137 @@
+        'Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+      ]
+    },
+    {
+      imageSource: './images/blog-2.jpg',
+      imageAltText: 'A group of dogs all on leashes',
+      title: 'How to Walk Multiple Dogs',
+      paragraphs: [
+        'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.',
+        'Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+      ]
+    },
+    {
+      imageSource: './images/blog-3.jpg',
+      imageAltText: 'Black outline of a woman and a dog, with a red-orage sky in the background',
+      title: 'Teach Your Dog To Fetch',
+      paragraphs: [
+        'Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.',
+        'Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.'
+      ]
+    }
+  ]
+}
+
+addBlogs();
+setupFormHandling();
+
+function addBlogs() {
+  console.log('Creating blog elements');
+  const blogsToAdd = [];
+  for (let i = 0; i < allBlogData.blogs.length; i++) {
+    const blog = createBlog(allBlogData.blogs[i]);
+    blogsToAdd.push(blog);
+  }
+
+  console.log('Adding blog elements');
+  const section = document.getElementById('blog-page-content');
+  if (section !== null) {
+    for (let i = 0; i < blogsToAdd.length; i++) {
```

Comment:
You do loop through two things. You could probably simplify this add Blogs function to only call one loop (the blog data)